File: data/llama3_8B_ins_results/readable_format_HPEC/fortran/file260.f90
subroutine print_array(nx, ny, s, q)
implicit none

DATA_TYPE, dimension(ny) :: s
DATA_TYPE, dimension(nx) :: q
integer :: nx,ny
integer :: i
do i = 1, ny
write(0, DATA_PRINTF_MODIFIER) s(i)
if (mod(i - 1, 80) == 0) then
write(0, *)
end if
end do

do i = 1, nx
write(0, DATA_PRINTF_MODIFIER) q(i)
if (mod(i - 1, 80) == 0) then
write(0, *)
end if
end do
write(0, *)
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file260.cpp
#include <iostream>
#define DATA_TYPE int
#define DATA_PRINTF_MODIFIER "%d"
void print_array(int nx, int ny, DATA_TYPE s[], DATA_TYPE q[]) {
 for(int i=0; i<ny; i++) {
 printf(DATA_PRINTF_MODIFIER, s[i]);
 if(i%80 == 0) printf("\n");
 }

 for(int i=0; i<nx; i++) {
 printf(DATA_PRINTF_MODIFIER, q[i]);
 if(i%80 == 0) printf("\n");
 }

 printf("\n");
}
